rfc/rfc-0015-supply-chain-selection-through-traits.md

```diff
@@ -1,3 +1,157 @@
-[~~Draft PR~~](https://github.com/vmware-tanzu/cartographer/pull/)
+# Draft RFC <0000> <Title>
 
-[Draft Text](https://github.com/vmware-tanzu/cartographer/blob/rfc-0015-supply-chain-selection-through-traits/rfc/rfc-0015-supply-chain-selection-through-traits.md)
+## Summary
+
+A mechanism to allow a cluster to have multiple Supply Chains declared
+1. With similar purposes (or where one extends the behaviour of another)
+2. That will match the developer's requirements without the developer needing to understand naming permutations.
+
+
+```gherkin
+Given Supply Chains that represent permutations of a behaviour tree
+When A developer annotates their workload to select for a permutation
```

Comment:
Can we frame this as "When a developer describes characteristics of their workload". They shouldn't be "selecting" a permutation.